integration-testing/test/complete_connected/test_convergence.py

```diff
@@ -0,0 +1,7 @@
+import logging
+from tools.profiling import profile
+
+@profile
+def test_network_convergence(converged_complete_network):
+    logging.info("Network converged successfully")
+    pass
```

Comment:
What's the purpose of those `pass` statements?

Reply:
it's a NOOP

This is already merged but I'll remove it in a future commit